core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigLoader.java

```diff
@@ -33,6 +39,25 @@
    */
   void onDriverInit(@NonNull DriverContext context);
 
+  /**
+   * Triggers an immediate reload attempt.
+   *
+   * @return a stage that completes once the attempt is finished. If a configuration change was
+   *     detected, it's also guaranteed that internal driver components have been notified by that
+   *     time; note however that some react to the notification asynchronously, so they may not have
+   *     completely applied all resulting changes yet. If this loader does not support programmatic
+   *     reloading &mdash; which you can check by calling {@link #supportsReloading()} before this
+   *     method &mdash; the stage will fail immediately with an {@link
+   *     UnsupportedOperationException}.
+   */
+  @NonNull
+  CompletionStage<Void> reload();
```

Comment:
Would there be value in enclosing a `Boolean` value indicating whether or not the reload observed config changes, similar to how `TypeSafeDriverConfig#reload` returns a boolean if the config changed?  This may be a more direct way of understanding if the config has changed from a manual reload as opposed to registering for `ConfigChangeEvent`

Reply:
changes look good :+1: